staging/src/k8s.io/apiserver/pkg/server/options/encryptionconfig/plugins.go

```diff
@@ -26,8 +26,13 @@ import (
 	"github.com/golang/glog"
 
 	"k8s.io/apiserver/pkg/storage/value/encrypt/envelope"
+	"k8s.io/apiserver/plugin/pkg/kmsprovider/vault"
 )
 
+func init() {
+	KMSPluginRegistry.Register("vault", vault.KMSFactory)
```

Comment:
Please make this a real function that acceps the KMSPluginRegistry and call it from kube-apiserver.